source/includes/rest/_orders.md

```diff
@@ -503,7 +507,8 @@ will be automatically incremented by the total amount of the order.
     </tr>
     <tr>
       <td><code>upstream_id</code></td>
-      <td>Optional. A unique id for the order (generally the primary key generated by the order management system).</td>
+      <td>Optional. A unique id for the order (generally the primary key generated by the order management system).
+      Used to identify the order for updates.</td>
     </tr>
     <tr>
       <td><code>identifier</code></td>
```

Comment:
The descriptions for identifier and upstream_id overlap quite a bit. Is identifier the one that our users technically won't have access to in this version because of the `accepted` response? Also, since this is a combined endpoint, do we want to call out that provider and upstream_id are `"Required for updates."` or something like that?

Reply:
I wasn't sure about how to put those descriptions either. For Shopify (which was the model for these columns) `upstream_id` corresponds to the `id` on the order. `identifier` is where we store the `name` of the order. Shopify's docs have this super-helpful description for `name`: "The customer's order name as represented by a number."

The `identifier` is displayed in the UI on the purchase in the activity feed.

@bfortune347 Do you have thoughts on how to document these?

`Required for updates` sounds good. I can add that here an on refunds.

Reply:
Ian and I just chatted on this. Where we landed was that upstream_id is an **internal** ID used really as the primary key in the customer's system (like Ian mentions) whereas the identifier is an **external** ID that will be presented in the Drip UI, and should match what the customer's customer sees on their invoice or email confirmation, etc..

Those two values can be the same, but if they're not (like in Shopify's case), we have an option to differentiate.